src/ws-dropdown/dropdown-menu.js

```diff
@@ -206,7 +237,8 @@ export class DropdownMenu extends Component {
     const regex = new RegExp(this.state.filter, 'i');
     return this.state.items.filter(item => {
       // Don't show items which doesn't match the filter
-      if (this.props.filterable && this.state.filter && !regex.test(item.label)) {
+      if (this.props.filterable && this.state.filter && !regex.test(
+          item.label)) {
```

Comment:
Pls configure your IDE for 120 characters.

If it is still to long pls you should place the conditions in variables to keep the if readable.